core/dbt/parser/schema_test_builders.py

```diff
@@ -243,6 +247,20 @@ def extract_test_args(test, name=None) -> Tuple[str, Dict[str, Any]]:
     def severity(self) -> str:
         return self.modifiers.get('severity', 'ERROR').upper()
 
+    def tags(self) -> List[str]:
+        tags = self.modifiers.get('tags', [])
+        if not isinstance(tags, list):
```

Comment:
We currently support model-level tags specified as a standalone string instead of as a list of strings. If the user supplies a standalone string, can we wrap it in a list?